### PR TITLE
Banner close & breadcrumb url change

### PR DESCRIPTION
**Pull request by CallMeSlinky**
Here are two changes. I'm not sure if the breadcrumb for "/" is intended to be like that, but I thought it'd make more sense to actually view all the products. I also added a close button to the banner but this change maybe controversial.  😁

### CODE REVIEW

File: src/ui/banner.tsx [hunk 1]
```diff
@@ -1,19 +1,44 @@
...
+import { X } from "lucide-react";
+import { useState } from "react";
 import { YnsLink } from "./yns-link";
 
 export const Banner = () => {
+	const [isOpen, setIsOpen] = useState(true);
+
+	const handleClose = () => {
+		setIsOpen(false);
+	};
```
Comment (zaiste) [inline src/ui/banner.tsx:12]: @CallMeSlinky could you add early `return` instead of the `isOpen && ...` syntax?

**Review by zaiste — CHANGES_REQUESTED** [review]

**Comment by zaiste** [comment]
@CallMeSlinky thanks for the contribution! the banner is temporary but we can add it to the codebase as an example

**Comment by zaiste** [comment]
@CallMeSlinky great! 

just a nitpick: using `git rebase` on a PR with `main` won’t create those `Merge branch ...` commits.

**Comment by CallMeSlinky** [comment]
> @CallMeSlinky great!
> 
> just a nitpick: using `git rebase` on a PR with `main` won’t create those `Merge branch ...` commits.

I see, sorry! I'm quite new to contributing on Github. I shall read up on this more, as it's quite daunting! 😨

**Comment by zaiste** [comment]
@CallMeSlinky no worries! you did great!

**Pull request merged**